datasets/mmarco/mmarco.py

```diff
@@ -0,0 +1,100 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""mMARCO dataset."""
+
+import datasets
+
+
+_CITATION = """
+@misc{bonifacio2021mmarco,
+      title={mMARCO: A Multilingual Version of the MS MARCO Passage Ranking Dataset},
+      author={Luiz Henrique Bonifacio and Israel Campiotti and Vitor Jeronymo and Roberto Lotufo and Rodrigo Nogueira},
+      year={2021},
+      eprint={2108.13897},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """
+mMARCO translated datasets
+"""
+
+_V1_BASE_URLS = {
+    "collections": "https://huggingface.co/datasets/unicamp-dl/mmarco/resolve/main/data/v1.1/collections/",
+    "train": "https://huggingface.co/datasets/unicamp-dl/mmarco/resolve/main/data/v1.1/queries/train/",
+    "dev": "https://huggingface.co/datasets/unicamp-dl/mmarco/resolve/main/data/v1.1/queries/dev/",
+}
+
+
+class MsMarcoConfig(datasets.BuilderConfig):
+    """BuilderConfig for MS MARCO."""
+
+    language: str = "english"
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for MS MARCO
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(MsMarcoConfig, self).__init__(**kwargs)
```

Comment:
No need to create a specific `datasets.BuilderConfig` subclass if you are not creating additional config attributes, others than name, description, version,...